site/bps/BP-template.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "BP-XYZ: capation of bookkeeper proposal"
+issue: https://github.com/apache/bookkeeper/<issue-number>
+state: "one of ['Under Discussion', 'Accepted', 'Adopted', 'Rejected']"
```

Comment:
Adopted is 'merged fully to master'? Or shall we have something like' Development in progress', 'partially implemented'...
Users will see many things now , so it must be clear that not all the proposed stuff is on master or released

Reply:
yes 'Adopted' means it is done, implementation is merged and documentation is available. It can be announced for a given release. BP is intended for "design doc", not for task tracking. The task tracking should be just done in github issues.

Reply:
ok